app/build.gradle

```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
+    compileSdkVersion 27
     buildToolsVersion '26.0.2'
 
     defaultConfig {
         applicationId "com.evanfuhr.pokemondatabase"
         minSdkVersion 23
-        targetSdkVersion 25
+        targetSdkVersion 27

```

Comment:
What motivated this change? I agree with it, but curious why.

Reply:
Just keeping it up to date.